src/core/tests/type_prop/fft_base_complex.cpp

```diff
@@ -0,0 +1,514 @@
+                                                       {10, 11, ov::no_label, 13}},
+        FFTConstantAxesAndConstantSignalSizeTestParams{{Dimension(0, 18), 180, Dimension(0, 400), 2},
+                                                       {2},
+                                                       {2},
+                                                       {87, 180, 390, 2},
+                                                       {2, 0},
+                                                       {390, 87},
+                                                       {ov::no_label, 11, ov::no_label, 13}},
+        FFTConstantAxesAndConstantSignalSizeTestParams{{Dimension(8, 129), 50, 130, Dimension(0, 500), 2},
+                                                       {3},
+                                                       {3},
+                                                       {Dimension(8, 129), 40, 130, 600, 2},
+                                                       {3, 0, 1},
+                                                       {600, -1, 40},
+                                                       {10, ov::no_label, 12, ov::no_label, 14}}};
+};
+
+TYPED_TEST_SUITE_P(FFTConstantAxesAndConstantSignalSizeTest);
+
+TYPED_TEST_P(FFTConstantAxesAndConstantSignalSizeTest, constant_axes_and_signal_size) {
+    for (auto params : this->test_params) {
+        set_shape_labels(params.input_shape, 10);
+        auto data = std::make_shared<op::v0::Parameter>(element::f32, params.input_shape);
+        auto axes_input = op::v0::Constant::create<int64_t>(element::i64, params.axes_shape, params.axes);
+
+        std::shared_ptr<TypeParam> dft;
+        if (params.signal_size.empty()) {
+            dft = std::make_shared<TypeParam>(data, axes_input);
+        } else {
+            auto signal_size_input =
+                op::v0::Constant::create<int64_t>(element::i64, params.signal_size_shape, params.signal_size);
+            dft = std::make_shared<TypeParam>(data, axes_input, signal_size_input);
+        }
+
+        EXPECT_EQ(dft->get_element_type(), element::f32);
+        EXPECT_EQ(dft->get_output_partial_shape(0), params.ref_output_shape);
+        if (params.expected_labels.empty()) {
+            EXPECT_EQ(get_shape_labels(dft->get_output_partial_shape(0)), get_shape_labels(params.input_shape));
+        } else {
+            EXPECT_EQ(get_shape_labels(dft->get_output_partial_shape(0)), params.expected_labels);
+        }
+    }
+}
+
+REGISTER_TYPED_TEST_SUITE_P(FFTConstantAxesAndConstantSignalSizeTest, constant_axes_and_signal_size);
+INSTANTIATE_TYPED_TEST_SUITE_P(type_prop, FFTConstantAxesAndConstantSignalSizeTest, FFTBaseTypes);
+
+struct NonConstantAxesTestParams {
+    PartialShape input_shape;
+    PartialShape axes_shape;
+    PartialShape ref_output_shape;
+};
+
+template <class TOp>
+class FFTNonConstantAxesTest : public TypePropOpTest<TOp> {
+public:
+    std::vector<NonConstantAxesTestParams> test_params{
```

Comment:
Make it `static const`?
Check similar usage.

Reply:
Updated to `const`, `static const` causes compilation error.